tests/integration/deleteIndex.test.ts

```diff
@@ -0,0 +1,33 @@
+import { utils } from "@pinecone-database/pinecone";
+import { getPineconeClient } from "@src/pinecone.js";
+import { deleteIndex } from "@src/deleteIndex.js";
+
+describe("Delete", () => {
+  const INDEX_NAME = "test-index-for-delete";
```

Comment:
Randomize the index name so that the timeout hack isn't needed.

Reply:
I've created [little test util fn](https://github.com/pinecone-io/semantic-search-example/pull/3/commits/1004a43cd6f04a350ccf0327e42e32e7e97783f5#diff-a5602548f1309e2f04ddade752e13db9577192a9025d018ca068b7064b9094eb) to handle this.